doc/workshop/SparseSensing/exercises/testSPSLOptiTwist.xml

```diff
@@ -18,24 +18,24 @@
   </RunInfo>
 
   <Files>
-    <Input name="twistFile" type=""></Input><!-- Provide the csv file that contains the names of the samples files-->
-    <Input name="refDO" type=""></Input><!-- Provide the csv of the case onwhich you want to show the sensors i.e., maximum initial power (350 W) and maximum initial Temperature-->
+    <Input name="twistFile" type="">../../data/Perturbations.csv</Input><!-- Provide the csv file that contains the names of the samples files-->
+    <Input name="refDO" type="">../../data/350_330.csv</Input><!-- Provide the csv of the case onwhich you want to show the sensors i.e., maximum initial power (350 W) and maximum initial Temperature-->
   </Files>
 
   <Steps>
     <IOStep name="LoadCSV">
       <Input class="Files" type="">twistFile</Input>
-      <Input class="Files" type=""></Input><!--Fill in-->
+      <Input class="Files" type="">refDO</Input><!--Fill in-->
       <Output class="DataObjects" type="HistorySet">TwistDO</Output>
       <Output class="DataObjects" type="PointSet">myDO</Output>
     </IOStep>
     <PostProcess name="mySPpp">
       <Input class="DataObjects" type="HistorySet">TwistDO</Input>
-      <Model class="Models" type="PostProcessor"></Model><!--Fill in-->
+      <Model class="Models" type="PostProcessor">mySPSL</Model><!--Fill in-->
       <Output class="DataObjects" type="DataSet">outPP</Output>
     </PostProcess>
     <IOStep name="print">
-      <Input class="DataObjects" type="DataSet">outPP</Input>
+      <Input class="DataObjects" type="DataSet">outPP</Input>cd 
```

Comment:
Is `cd` a typo?

Reply:
Yes! Thank you for catching that. I have removed cd.